docs/section19/19-20.md

```diff
@@ -0,0 +1,73 @@
+
+            - Get the total supply cfm, need to get the first element in the list of fan CFMs: `supply_cfm_b = fan_sys_CFM[0]`  
+            - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                    - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                    - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`     
+                - Get the analogous zone object in the P_RMI: `zone_p = match_data_element(P_RMR,Zone,zone_b)` 
+                - Check if the zone has mechanical cooling in the proposed: `If (zone_p.non_mechanical_cooling_fan_airflow != Null AND zone_p.non_mechanical_cooling_fan_airflow != 0): zones_served_by_BL_hvac_has_non_mech_cooling = True`  
+                - Get list of hvac systems serving this zone in the proposed design model: `hvac_sys_list_p = Get_list_hvac_systems_associated_with_zone(P_RMI, zone_p)`  
+                - Check if any of the hvac systems serving the zone have non-mechanical cooling as the cooling_system type: `for hvac_p in hvac_sys_list_P:`  
+                    - Check if the system has non-mechanical cooling: `if hvac_p.cooling_system.cooling_system_options == "NON_MECHANICAL": zone_hvac_in_PR_has_non_mech_cooling = True`  
+
+            - Calculate W/CFM: `fan_power_W_CFM = total_fan_power/supply_cfm_b`                 
+            
+            - **Rule Assertion:** 
+            - Case 1: If fan_power_W_CFM equals 0.3: `if zone_hvac_in_PR_has_non_mech_cooling == False and zones_served_by_BL_hvac_has_non_mech_cooling == False and fan_power_W_CFM == 0.3: outcome = "PASS"`  
+            - Case 2: Elif conservative comparison less equal:`elif AHJ_RA_compare == True and fan_power_W_CFM <= 0.3: PASS`  
+            - Case 3: Elif there is non-mechanical cooling associated with the zones served by the baseline HVAC system (should just be one) in the proposed model then the outcome is UNDETERMINED: `if zone_hvac_in_PR_has_non_mech_cooling ==  True or zones_served_by_BL_hvac_has_non_mech_cooling ==  True: outcome = UNDETERMINED and raise_message "<Insert hvac.id> has zone(s) with non-mechanical cooling in the proposed design, conduct a manual check that the baseline building design includes a fan power allowance of Pfan = CFMnmc × 0.054, where, CFMnmc = the baseline non-mechanical cooling fan airflow, cfm for the non-mechanical cooling fan in additional to the 0.3 W/CFM allowance for the HVAC system."`  
```

Comment:
We have zone.non_mecahnical_cooling_fan_power and zone.non-mecanical_cooling_fan_airflow. Why can't we check that it's 0.054 W/CFM? Or perhaps check that the total baseline fan power is equal to correct total W for all fans including non-mechanical cooling?

Reply:
I agree that 19-20 and 21 should be combined. Revised.